CONTRIBUTING.md

```diff
@@ -1,4 +1,37 @@
-Contributing to the LaunchDarkly SDK for PHP
-================================================
+# Contributing to the LaunchDarkly SDK FOR PHP
 
-We encourage pull-requests and other contributions from the community. We've also published an [SDK contributor's guide](http://docs.launchdarkly.com/docs/sdk-contributors-guide) that provides a detailed explanation of how our SDKs work.
\ No newline at end of file
+LaunchDarkly has published an [SDK contributor's guide](https://docs.launchdarkly.com/docs/sdk-contributors-guide) that provides a detailed explanation of how our SDKs work. See below for additional information on how to contribute to this SDK.
+
+## Submitting bug reports and feature requests
+ 
+The LaunchDarkly SDK team monitors the issue tracker associated with the `launchdarkly/php-server-sdk` SDK repository. Bug reports and feature requests specific to this SDK should be filed in this issue tracker. The SDK team will respond to all newly filed issues within two business days.
```

Comment:
In my other pull requests I linked to the issue tracker and then just said "... with the SDK repository" since the link made it clear to which SDK repository I was referring. It would be good to include a link here too.

Example: https://github.com/launchdarkly/go-server-sdk-private/pull/141/files#diff-6a3371457528722a734f3c51d9238c13R9